modules/api/src/main/java/org/eclipse/xpanse/api/exceptions/CommonExceptionHandler.java

```diff
@@ -60,20 +63,38 @@ public Response handleConstraintViolationException(ConstraintViolationException
     @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
     public Response handleRuntimeException(RuntimeException ex) {
         String failMessage = ex.getMessage();
-        return Response.errorResponse(ErrCode.RUNTIME_ERROR, failMessage);
+        return Response.errorResponse(ResultCode.RUNTIME_ERROR, failMessage);
     }
 
+    /**
+     * Exception handler for HttpMessageConversionException.
+     */
+    @ExceptionHandler({HttpMessageConversionException.class})
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    public Response handleHttpMessageConversionException(HttpMessageConversionException ex) {
+        ex.printStackTrace();
+        String failMessage = ex.getMessage();
+        return Response.errorResponse(ResultCode.BAD_PARAMETERS, failMessage);
+    }
+
+    /**
+     * Exception handler for EntityNotFoundException.
+     */
     @ExceptionHandler({EntityNotFoundException.class})
     @ResponseStatus(HttpStatus.NOT_FOUND)
     public Response handleNotFoundException(EntityNotFoundException ex) {
         String failMessage = ex.getMessage();
-        return Response.errorResponse(ErrCode.BAD_PARAMETERS, failMessage);
+        return Response.errorResponse(ResultCode.BAD_PARAMETERS, failMessage);
     }
 
+    /**
+     * Exception handler for Exception.
+     */
     @ExceptionHandler({Exception.class})
     @ResponseStatus(HttpStatus.NOT_FOUND)
     public Response handleException(Exception ex) {
+        ex.printStackTrace();

```

Comment:
I think we shouldn't keep this, just use a log instead.

Reply:
Sure. I will delete it.